### DISCUSSION TITLE
building on windows11 failed

### DISCUSSION THREAD

**Discussion by frank-s-liu** [WerWolv/ImHex#845] [Get Help] [answered]
I compiling ImHex project following with https://github.com/WerWolv/ImHex/blob/master/dist/compiling/windows.md
the compiled the Imhex git-commit is commit c8af25533d800f70d231da9ff6383c0fadfd32ec which is with tag of "tag: v1.25.0"
and also the  ImHex-Patterns is from tag ImHex-v1.25.0

![imhex build error](https://user-images.githubusercontent.com/70690476/206946685-b27c582d-b8f2-450f-92f4-4859aea6f6c0.png)

**Comment by frank-s-liu** [comment]
does any other configuration need to set?

**Comment by frank-s-liu** [comment]
clearly, it is the install path of msys2 is under "C:\Program Files\" which cointains "white space"

**Comment by WerWolv** [comment, marked as answer]
Yep. Install msys2 somewhere without spaces in the path. C:/msys2 for example or on an external drive
